packages/amplify-category-hosting/__tests__/lib/S3AndCloudFront/helpers/configure-Website.test.js

```diff
@@ -0,0 +1,35 @@
+const inquirer = require('inquirer');
+const mockirer = require('mockirer'); 
+
+const mockTemplate = require('../../../../__mocks__/mockTemplate');
+
+const configureWebsite = require('../../../../lib/S3AndCloudFront/helpers/configure-Website'); 
+
+describe('configure-Website', ()=>{
+    const mockContext = {
+        exeInfo: {
+            template: mockTemplate
+        }
+    }; 
+
+    const indexDoc = 'index.html'; 
+    const errorDoc = 'error.html'; 
+    beforeAll(()=>{
+        mockirer(inquirer, {
+            IndexDocument: indexDoc,
+            ErrorDocument: errorDoc
+        })
+    }); 
+
+    beforeEach(()=>{
+
+    }); 
+
+    test('configure', async ()=>{
+        const { WebsiteConfiguration } = mockContext.exeInfo.template.Resources.S3Bucket.Properties;
+        const result = await configureWebsite.configure(mockContext); 
+        expect(result).toEqual(mockContext); 
```

Comment:
Do we want to test default values used on prompt for `IndexDocument` and `ErrorDocument`

Reply:
default values are provided in the current template, not sure why that needed to be tested.

Reply:
the default values are the current values.